### PR TITLE
Book Suggestion Routes tests

### PR DESCRIPTION
**Pull request by mcolombo87**
### Summary
New Tests for Book Suggestions' Routes.
### ScreenShoots
image
### Trello
https://trello.com/c/swvfOsLw/28-crear-test-para-endpoint-de-creaci%C3%B3n-de-un-booksuggestion

### CODE REVIEW

File: spec/controllers/api/v1/book_suggestions_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+describe Api::V1::BookSuggestionsController, type: :controller do
+  describe 'POST #create' do
+    context 'When puts on a new Book Suggestion' do
+      let!(:bsuggestion_params) do
+        booksugg_autogenerate = build(:book_suggestion)
```
Comment (facucompe) [inline spec/controllers/api/v1/book_suggestions_controller_spec.rb]: Use `attributes_for(:book_suggestion)` instead

File: spec/controllers/api/v1/book_suggestions_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        expect(response_body).to eq(bsuggestion_params)
+      end
+      it 'responds with 201 status' do
+        expect(response).to have_http_status(:created)
+      end
+    end
+
+    context 'When try to insert a new Book Suggestion but some params are missing' do
+      let!(:bsuggestion_params) do
+        booksugg_autogenerate = build(:book_suggestion)
```
Comment (facucompe) [inline spec/controllers/api/v1/book_suggestions_controller_spec.rb]: Same here

**Review by facucompe — CHANGES_REQUESTED** [review]

File: spec/controllers/api/v1/book_suggestions_controller_spec.rb (hunk 2 above)
Comment (mcolombo87) [reply spec/controllers/api/v1/book_suggestions_controller_spec.rb]: Done!

**Review by floresv — APPROVED** [review]
LGTM

File: spec/controllers/api/v1/book_suggestions_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      end
+      it 'responds with 201 status' do
+        expect(response).to have_http_status(:created)
+      end
+    end
+
+    context 'When try to insert a new Book Suggestion but some params are missing' do
+      let!(:bsuggestion_params) do
+        booksugg_autogenerate = attributes_for(:book_suggestion)
+        booksugg_autogenerate.as_json(only: %i[price genre])
```
Comment (facucompe) [inline spec/controllers/api/v1/book_suggestions_controller_spec.rb:24]: `booksugg_autogenerate.delete!(:key)`

**Review by facucompe — APPROVED** [review]
Change that if you want

File: spec/controllers/api/v1/book_suggestions_controller_spec.rb (hunk 3 above)
Comment (mcolombo87) [reply spec/controllers/api/v1/book_suggestions_controller_spec.rb:24]: That ways is better than my way, I give you that, but I prefer don't modify this PR and move on!

**Pull request merged**